prow/scripts/lib/gcloud.sh

```diff
@@ -462,4 +462,14 @@ function gcloud::cleanup {
   fi
 }
 
+# gcloud::update_cluster_version_for_channel checks for latest possible version in GKE_RELEASE_CHANNEL and updates GKE_CLUSTER_VERSION accordingly
+# Required exported variables:
+# GKE_RELEASE_CHANNEL
+function gcloud::update_cluster_version_for_channel() {
```

Comment:
Can we change the name to something like 'set_latest_cluster_version_for_channel"
imho it's more self-explanatory

Reply:
I've changed the name